### DISCUSSION TITLE
materials -  iOS vs android

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#2974] [General]
Hi everyone, 

Today I purchased a Samsung A52s and compared it to my iPhone13 pro max.

I noticed there's some major differences in how materials are displayed on each device. Is there anyway to fix this issue

![android - Samsung A52s](https://user-images.githubusercontent.com/90825510/141115218-32451615-bc23-4adf-b20c-f5e857ec6a50.jpg)
![iPhone 13 pro Max](https://user-images.githubusercontent.com/90825510/141116663-e18d3351-2584-4e64-8316-956fb72c4b88.jpg)
)

**Comment by ghost** [comment]
This _looks_ to be an issue of light estimation which will be different on each device as they use different tech stacks and cameras. The top looks darker and like it is generating a lower res lighting data (probably Android?) but it looks appropriate for its understanding of reality. If you pan around the room that should help to build better lighting information. Can you describe which image is on which device, and whether it's SceneViewer, WebXR or QuickLook?

**Reply by Jasonp8000** [reply]
The top image is with scene-viewer on android and the bottom image is quick-look on iPhone 13 pro. 

Can scene viewer/WebXR produce shiny materials? Is there a work around for this issue ?

**Reply by elalish** [reply]
Yes, it's a well-known problem with ARCore's lighting estimation that it makes shiny things look matte (the reflection resolution is too low). WebXR mode will look  properly shiny because by default  it uses the  same lighting as the 3D mode (aside from a bug in v1.8 which was fixed in #2847). You  can see the same problem as scene viewer has by enabling `xr-environment`, which is why it's opt-in.
